packages/client3/components/design-system/input/radobutton.stories.tsx

```diff
@@ -0,0 +1,18 @@
+import { ComponentMeta, ComponentStory } from "@storybook/react";
+
+import { RadioButton } from "./radiobutton";
+
+export default {
+  component: RadioButton,
+  title: "Components/Input/RadioButton",
+} as ComponentMeta<typeof RadioButton>;
+
+export const RadioButtonStory: ComponentStory<typeof RadioButton> = (args) => (
+  <RadioButton {...args} />
+);
+
+RadioButtonStory.args = {
+  name: "rb",
+  disabled: false,
+  checked: false,
+};
```

Comment:
PS, rename the file please xD 

'radobutton'

Reply:
Lol, renamed it